vpr/src/route/rr_graph_area.cpp

```diff
@@ -167,7 +167,7 @@ void count_bidir_routing_transistors(int num_switch, int wire_to_ipin_switch, fl
                 num_edges = device_ctx.rr_nodes[from_node].num_edges();
 
                 for (iedge = 0; iedge < num_edges; iedge++) {
-                    RRNodeId to_node = RRNodeId(size_t(rr_graph.edge_sink_node(RRNodeId(from_node), iedge)));
+                    RRNodeId to_node = RRNodeId(rr_graph.edge_sink_node(RRNodeId(from_node), iedge));
```

Comment:
Typecast to `RRNodeId` is not needed now as `edge_sink_node` return type is `RRNodeId`.